src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,58 @@
+# Frame V4 API FAQ
+
+The Frame V4 API allows developers to integrate Frame.io’s surface for creatives and their teams to manage their work for every step of the asset delivery lifecycle from plan to deliver.
```

Comment:
```suggestion
The Frame V4 API allows developers to integrate Frame.io’s surface for creatives and their teams to manage their work for every step of the asset delivery lifecycle from plan to delivery.
```
Personally, I don't think we need to try to summarize what the API can be used for in this document, but I'll defer to marketing and product folks for their best judgement on that front 😄

Reply:
I took this from marketing and hear you so I'll just shorten it up.